utils/build-tooling-libs

```diff
@@ -1,18 +1,18 @@
 #!/usr/bin/env python
-# utils/build-parser-lib - Helper tool for building the parser library -*- python -*-
+# utils/build-tooling-libs - Helper tool for building the parser and swift-static-mirror libraries -*- python -*-
```

Comment:
Since you are already at it, could you change this to clarify we are talking about the SwiftSyntax parser and not the “real” parser?

```suggestion
# utils/build-tooling-libs - Helper tool for building the SwiftSyntax parser and swift-static-mirror libraries -*- python -*-
```